python/tools/dht/tests.py

```diff
@@ -371,6 +478,9 @@ def _gottaGetThemAllPokeNodes(self, consumer, hashes, nodes=None):
                 for n in DhtFeatureTest.foreignNodes:
                     nodes.add(n)
 
+    def __normalConditionTest(self):
```

Comment:
This is odd since `DhtFeatureTest` inherits from `FeatureTest`, so it has `self._normalConditionTest`. Therefor, at line 574, the call is simply:
```python
self._normalConditionTest()
```
as before except that the name of the function changed. In short, `__normalConditionTest` is useless.